### DISCUSSION TITLE
Giscus: เอา GitHub Discussion มาใส่ใน Blog ตัวเองง่ายๆ | First Sutham

### DISCUSSION THREAD

**Discussion by giscus** [heyfirst/heyfirst.co#9] [General]
# Giscus: เอา GitHub Discussion มาใส่ใน Blog ตัวเองง่ายๆ | heyfirst.co

เอา GitHub Discussion มาใช้สร้าง Comment box บน Blog ตัวเองง่าย ๆ ด้วย Giscus

http://localhost:3000/blog/introducing-giscus

**Comment by heyfirst** [comment]
สวัสดีฮะทุกคนนนนนน! 

![](https://camo.githubusercontent.com/e35921e2d0900268f4294cca4037e786b061a111f3fa4ffc9ac2d3fa754cf4aa/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4d54636c6643723474566769732f67697068792e676966)

**Reply by nutchy** [reply]
แจ่มมมมม

**Reply by heyfirst** [reply]
มันแจ้งเตือนในอีเมลด้วยว่ะ 🔥 โคตรเจ๋งงงง

**Comment by dtinth** [comment]
สมัยก่อนเคยใช้ Utterances เพิ่งรู้ว่าเดี๋ยวนี้มี Giscus ด้วย เจ๋งงงงง

**Reply by heyfirst** [reply]
🙆🏻🙆🏻 เย้ ผมก็ลองหาดู บังเอิญเจอพอดีเฉยๆ เลยฮะพี่ แต่ star ยังน้อยเมื่อเทียบกับ utterances ฮะ
